doc/classes/VisualShaderNode.xml

```diff
@@ -11,6 +11,7 @@
 			<return type="Array">
 			</return>
 			<description>
+				Returns an [Array] containing default values for all of the input ports of the node in the form {index, value, index, value}.
```

Comment:
`{ }` is what we use for dictionaries, so it's confusing here.
If you mean an array of 4 elements, it should be:
```
[code][index, value, index, value][/code]
```
or if you want to signify that the number of values depends on the number of input ports defined, you could do:
```
[code][index, value, index, value, ...][/code]
```